docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,325 @@
 # Registering a Workload Execution Cluster
 
-This document will tell users what how to register a WEC in an ITS.
+This document explains how to register a Workload Execution Cluster (WEC) with an Inventory and Transport Space (ITS) in KubeStellar.
 
-TODO: write this.
+## Overview
+
+Registering a WEC in an ITS is the same process as registering a managed cluster with an OCM hub cluster. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+**Terminology Mapping:**
+- **OCM Hub** → **KubeStellar ITS** (Inventory and Transport Space)
+- **OCM Managed Cluster** → **KubeStellar WEC** (Workload Execution Cluster)
+- **OCM Agent** → **OCM Agent** (same component, installed on WEC)
```

Comment:
This did not render as you intended.

<img width="782" alt="image" src="https://github.com/user-attachments/assets/3b53f651-a814-48a9-8753-21e580b5464b" />

Using a proper section heading would probably fix it.